src/Shared/UTXOSelectorModal.razor

```diff
@@ -216,20 +219,28 @@
         if (e.Value is not List<UTXO>) return;
         var selectedUTXOsValue = ((List<UTXO>)e.Value).Sum(x => ((Money)x.Value).ToUnit(MoneyUnit.BTC));
         if (selectedUTXOsValue == 0) return;
-        var minimumChannelValue = new Money(Constants.MINIMUM_CHANNEL_CAPACITY_SATS).ToUnit(MoneyUnit.BTC);
-        var maxChannelRegtestValue = new Money(Constants.MAXIMUM_CHANNEL_CAPACITY_SATS_REGTEST).ToUnit(MoneyUnit.BTC);
-        if (selectedUTXOsValue < minimumChannelValue)
+        decimal minimumValue;
+        if (IsWalletWithdrawalValidation)
         {
-            e.ErrorText = $"The combined amount of the UTXOs selected must be greater than {minimumChannelValue:f8} BTC";
+             minimumValue = Constants.MINIMUM_WITHDRAWAL_BTC_AMOUNT;
+        }
+        else
+        {
+             minimumValue = new Money(Constants.MINIMUM_CHANNEL_CAPACITY_SATS).ToUnit(MoneyUnit.BTC);
+        }
+             var maxChannelRegtestValue = new Money(Constants.MAXIMUM_CHANNEL_CAPACITY_SATS_REGTEST).ToUnit(MoneyUnit.BTC);
+        if (selectedUTXOsValue < minimumValue)
+        {
+            e.ErrorText = $"The combined amount of the UTXOs selected must be greater than {minimumValue:f8} BTC";
             e.Status = ValidationStatus.Error;
         }
-        else if (selectedUTXOsValue > maxChannelRegtestValue && network == Network.RegTest)
+        else if (selectedUTXOsValue > maxChannelRegtestValue && network == Network.RegTest && IsWalletWithdrawalValidation)
```

Comment:
This should be `!IsWalletWithdrawalValidation` I think

Reply:
I've updated the code. Looks like old PR.

Reply:
I think this one hasn't change